--- eet-1.5.0/src/examples/Makefile.am~	2011-11-18 04:08:20.000000000 +0100
+++ eet-1.5.0/src/examples/Makefile.am	2012-01-19 09:54:13.300469591 +0100
@@ -8,7 +8,7 @@ AM_CPPFLAGS = \
 -I$(top_srcdir)/src/lib \
 @EINA_CFLAGS@
 
-pkglib_PROGRAMS = eet_basic eet_file eet_data_simple eet_data_nested eet_data_file_descriptor_01 eet_data_file_descriptor_02 eet_data_cipher_decipher
+pkglibexec_PROGRAMS = eet_basic eet_file eet_data_simple eet_data_nested eet_data_file_descriptor_01 eet_data_file_descriptor_02 eet_data_cipher_decipher
 
 eet_basic_SOURCES = eet-basic.c
 eet_basic_LDADD = $(top_builddir)/src/lib/libeet.la @EINA_LIBS@
